lang/src/org/partiql/lang/ast/StatementToExprNode.kt

```diff
@@ -27,6 +27,28 @@ internal typealias IonElementMetaContainer = com.amazon.ionelement.api.MetaConta
 fun Statement.toExprNode(ion: IonSystem): ExprNode =
     StatementTransformer(ion).transform(this)
 
+fun Expr.toExprNode(ion: IonSystem): ExprNode {
+    return StatementTransformer(ion).transform(this)
+}
+
+fun SetQuantifier.toSetQuantifier(): org.partiql.lang.ast.SetQuantifier =
+    when (this) {
+        is SetQuantifier.All -> org.partiql.lang.ast.SetQuantifier.ALL
+        is SetQuantifier.Distinct -> org.partiql.lang.ast.SetQuantifier.DISTINCT
+    }
+
+fun com.amazon.ionelement.api.MetaContainer.toPartiQlMetaContainer(): org.partiql.lang.ast.MetaContainer {
+    val nonLocationMetas: List<Meta> = this.values.map {
+        // We may need to account for this in the future, but for now we require that all metas placed
+        // on any `partiql_ast` instances to implement Meta.  It's not clear how to deal with that now
+        // so we should wait until it's needed.
+        val partiQlMeta = it as? Meta ?: error("The meta was not an instance of Meta.")
+        partiQlMeta
+    }
+
+    return org.partiql.lang.ast.metaContainerOf(nonLocationMetas)
+}
+
```

Comment:
May need some guidance on a better approach to convert a PIG `CallAgg` to an ExprNode `CallAgg` without exposing these methods.